src/ui/story.js

```diff
@@ -47,24 +47,26 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             // ],
             // translationOf: null,
         },
-        modelRelay: [{
-            target: "contentString",
-            singleTransform: {
-                type: "sjrk.storyTelling.transforms.arrayToString",
-                input: "{that}.model.content",
-                separator: ". ",
-                stringOnly: true,
-                path: "contentString"
+        modelRelay: {
+            contentToContentString: {
+                source: "{that}.model.content",
+                target: "contentString",
+                singleTransform: {
+                    type: "sjrk.storyTelling.transforms.arrayToString",
+                    delimiter: ". ",
```

Comment:
This is causing an interesting "bug". 

1. Add a text block with text like "Some Text!"
2. Add another text block with text like "more text"
3. The output is "Some Text!. more text"; Notice the double punctuation "!.".

I imagine that what happens with that will depend on the synthesizer used. Some might ignore it, others might read it as "dot". If you end up using the contentString for output the text visually, it will look strange and be incorrect. Also what about other languages that have different punctuation? It may be better not to try to add punctuation and just separate based on " ". What do you think?

Reply:
It certainly isn't an ideal situation one way or the other. If I introduce some special case coding to check for grammatical incompatibility with the delimiter, then we're making this transform function less general and therefore less likely to be able to be merged into Infusion in the future. If we don't, then as you've described we risk tripping up certain screen readers or other AT's.

If we set the delimiter to `" "` then we won't get the pause between blocks, and the synthesizer may continue reading the next one as if it's all one sentence:
1. Add a text block with text like "Some text"
2. Add another text block with text like "more text"
3. the output is "Some text more text", which will be read without pauses and could cause confusion

I'm not sure what the best approach is, here. Perhaps we can file a new Jira to capture this issue and have a discussion there, and scope it to make the necessary changes to detect and compensate for punctuation at the end of a term that would be incompatible with the delimiter.

Reply:
Sure, file a new JIRA. I'm starting to wonder more if you should be storing the contentString in the model at all. Perhaps even just using the Orator component to read through the content in the story instead. Not sure if this would solve the issue of pausing between "unfinished" blocks, but it might help with other things like the empty "by" and "keywords" sections. Just throwing some ideas out there for the new JIRA.

Reply:
[SJRK-271](https://issues.fluidproject.org/browse/SJRK-271) has been filed to capture this